### PR TITLE
Section 2 - Jason Burks

### PR DESCRIPTION
**Pull request by jasonburks**
- wrote lesson zero test

### CODE REVIEW

File: test/01LessonOneTest.js [hunk 1]
```diff
@@ -5,36 +5,55 @@ var assert = require('assert');
...
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
+    if (big > small) {
+      return true;
+    } else {
+      return false
+    }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js]: FYI, you could just return (big > small)

File: test/01LessonOneTest.js [hunk 2]
```diff
@@ -5,36 +5,55 @@ var assert = require('assert');
...
 
 function lessThan(small, big) {
     // should return true if the first argument is less than the second argument
     // and return false if the first argument is greater than the second argument
-
+    if (small < big) {
+      return true;
+    } else {
+      return false
+    }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js]: FYI, you could just `return (small < big)`

File: test/01LessonOneTest.js [hunk 3]
```diff
@@ -5,36 +5,55 @@ var assert = require('assert');
...
 function equalTo(a, b) {
     // should return true if the first argument is equal to the second argument
     // and should return true if the first argument is equal to the second argument
-
+    if (a === b) {
+      return true;
+    } else {
+      return false
+    }
 }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js:34]: FYI, you could just `return (a === b)`

File: test/01LessonOneTest.js [hunk 4]
```diff
@@ -5,36 +5,55 @@ var assert = require('assert');
...
 
 function notEqualTo(a, b) {
     // should return true if the first argument is not equal to the second argument
     // and should return false if the first argument is equal to the second argument
-
+    if (a !== b) {
+      return true;
+    } else {
+      return false
+    }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js]: FYI, you could just `return (a !== b)`

**Comment by jasonburks** [comment]
Made updates to address the comments.

File: katas/00KataZero.js [hunk 5]
```diff
@@ -6,6 +6,7 @@ var assert = require('assert');
 
 function sayHello() {
     // Your code here
+    return ('Hello!');
 
 }
```
Comment (paulrenenichols) [inline katas/00KataZero.js:11]: Awesome!

File: test/02LessonTwoTest.js [hunk 6]
```diff
@@ -8,11 +8,24 @@ function whichSpecies(character) {
...
 
 function isEven(number) {
     // should return true is number is even (divisible by 2)
-
+    if (number % 2 === 0) {
+      return true;
+    } else {
+      return false;
+    }
 }
```
Comment (paulrenenichols) [inline test/02LessonTwoTest.js:30]: @jasonburks Great work! Really appreciate that you're indenting the code properly!

File: test/04LessonFourTest.js [hunk 7]
```diff
@@ -4,38 +4,53 @@ var assert = require('assert');
 
 function arrayPop(arr) {
     // should remove the last item in the array, then return the arr
-
+    var poppedItem = arr.pop();
```
Comment (paulrenenichols) [inline test/04LessonFourTest.js:7]: Technically, you don't need to declare this variable if you never use it.

Same goes for the functions that follow.

Good job, though.

File: test/05LessonFiveTest.js [hunk 8]
```diff
@@ -4,20 +4,36 @@ var assert = require('assert');
...
     // divisible by 3 are replaced by the string "fizz", numbers divisible by 5
     // are replaced by the string "buzz", and the numbers divisble by both 3 and
     // 5 are replaced by fizzbuzz
-
+    var array = [ ];
+
+    for (var i = 0; i < 100; i++) {
+      var fizz = i % 3 == 0;
+      var buzz = i % 5 == 0;
+      array.push (fizz ? buzz ? "fizzbuzz" : "fizz" : buzz ? "buzz" : i);
```
Comment (paulrenenichols) [inline test/05LessonFiveTest.js:34]: Intense, man!

File: apps/05Mastermind.js [hunk 9]
```diff
@@ -6,7 +6,7 @@ var prompt = require('prompt');
 prompt.start();
 
 var board = [];
-var solution = '';
+var solution = 'abcd';
```
Comment (paulrenenichols) [inline apps/05Mastermind.js:9]: This isn't good.  This is resulting in a long solution.  Your program did this when I played it:

```
You ran out of turns! The solution was abcdfhbc
```

File: apps/07jquery-tic-tac-toe/script.js [hunk 10]
```diff
@@ -1,5 +1,59 @@
 'use strict';
 
 $(document).on('ready', function() {
-    // Put app logic in here
+  // Put app logic in here
+  var playerTurn = 'X';
+
+  $('#reset').click(function() {
```
Comment (paulrenenichols) [inline apps/07jquery-tic-tac-toe/script.js:7]: Pretty solid, @jasonburks!

File: apps/08jquery-towers-of-hanoi/script.js [hunk 11]
```diff
@@ -2,4 +2,34 @@
...
   // Put app logic here
-}
+  var allStacks = 'div[data-stack]';
+  var block = null;
+
+  function checkForWin(){
+    if (($(this).data('stack') > 1) && ($(this).children().length) === 4){
+      $('#announce-game-won').text('You Won!');
+    }
+  }
```
Comment (paulrenenichols) [inline apps/08jquery-towers-of-hanoi/script.js:12]: @jasonburks Your `checkForWin()` doesn't work. `this` is not defined when you call it.

**Comment by paulrenenichols** [comment]
@jasonburks Where is Lesson 10, Ajax?

**Pull request closed without merging**